charts/jenkins/templates/jenkins-backup-rbac.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
     "app.kubernetes.io/managed-by": "{{ .Release.Service }}"
     "app.kubernetes.io/instance": "{{ .Release.Name }}"
     "app.kubernetes.io/component": "{{ .Values.master.componentName }}"
+  {{- if .Values.backup.annotations }}
```

Comment:
Shouldn’t this be present in values.yaml as well? So that users can discover it (commented out)

Reply:
```
backup:
  annotations: {}
    # Example for authorization to AWS S3 using kube2iam
    # Can also be done using environment variables
    # iam.amazonaws.com/role: "jenkins"
```
Hi @timja, its already added there.

Reply:
Added an example for IRSA usage in the annotations:
```
annotations: {}
    # Example for authorization to AWS S3 using kube2iam or IRSA
    # Can also be done using environment variables
    # iam.amazonaws.com/role: "jenkins"
    # "eks.amazonaws.com/role-arn": "arn:aws:iam::123456789012:role/jenkins-backup"```